### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN001.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#47] [General]
# https://blog.3place.cn/3place/lmyn/LMYN001.html

https://blog.3place.cn/3place/lmyn/LMYN001.html

**Comment by picklecai** [comment]
《罗摩衍那》最初决不会出于一人之手。它最初一定是老百姓的创作，只是口头流传。

能流传的故事，一定有它的妙处。

**Reply by hbaolong** [reply]
也可能，只是因为当时的文化作品太少了。

**Comment by picklecai** [comment]
《罗摩衍那》：梵语，Rāmāyaṇa，意思为“罗摩的历险经历”。
